.github/pull_request_template.md

```diff
@@ -47,13 +47,12 @@ Describe in short the main changes with the new release.
 
 _Put an `x` in the boxes that apply._
 
-- [ ] I have read the [CONTRIBUTING](https://github.com/fetchai/agents-aea/blob/master/CONTRIBUTING.md) doc
-- [ ] I am making a pull request against the `main` branch (left side), from `develop`
-- [ ] Lint and unit tests pass locally and in CI
-- [ ] I have checked the fingerprint hashes are correct by running (`scripts/generate_ipfs_hashes.py`)
-- [ ] I have regenerated the latest API docs
-- [ ] I built the documentation and updated it with the latest changes
-- [ ] I have added an item in `HISTORY.md` for this release
+- [ ] I am making a pull request against the `main` branch from `develop`.
```

Comment:
I would be in favor of moving this release checklist somewhere so that it is not included in every PR.